static/js/app.js

```diff
@@ -694,8 +694,54 @@ function toggleClass(elem) {
       item.classList.remove("is-active");
   });
   elem.target.classList.add("is-active");
+
+  const findClassName = elem.target.classList;
+  const accordionClassArray = Array.from(findClassName);
+
+  const urlRelay = window.location.href;
+  const accordionHash = window.location.hash;
+
+  if (accordionHash === ""){
+    window.location.href = urlRelay.concat("#" + accordionClassArray[1]);
+  }
+
+  else {
+    window.location.href = urlRelay.replace(accordionHash, "#" + accordionClassArray[1]);
+  }
 }
```

Comment:
You can drop all the manipulation of `window.location.href` and manually adding the `#`. By setting a string as the `location.hash`, you get that functionality for free! 

```suggestion
  window.location.hash = accordionClassArray[1];
}
```

Additionally, how you're querying the class/hash ID (the second class name in the array) is really fragile. I would propose moving (or cloning) the class names in a `data-use-case` element in the [`use-cases.html`](https://github.com/mozilla/fx-private-relay/blob/anchor-links-accordion/privaterelay/templates/newlanding/a/use-cases.html) file. That way you're being more exact.

Reply:
Thanks for the suggestion, I also felt that it was a fragile approach. I looked into using a data attribute and used that method instead!